stylesheets/scss/style.scss

```diff
@@ -15,9 +15,11 @@
 // Fonts, mixins, variables, etc
 @import 'includes/fonts';
 @import 'includes/mixins';
+@import 'includes/functions';
```

Comment:
@nmwd  What is your thought on putting functions before mixins?  Going forward, there could be potential for functions to be used in mixins.